Update azure-storage-linux-container-pivot.md (#127420)
Azure/azure-content: articles/app-service/includes/configure-azure-storage/azure-storage-linux-container-pivot.md

### Troubleshooting

- The mount directory in the custom container should be empty. Any content stored at this path is deleted when the Azure Storage is mounted, if you specify a directory under */home*, for example. If you migrate files for an existing app, make a backup of the app and its content before you begin.

> - When mounting an NFS share, you will need to ensure that Secure Transfer Required is disabled on the storage account. App Service does not support mounting NFS shares when this is enabled. It uses port 2409 and Vnet integration and private endpoints as the security measure.

Reviewer: ```suggestion
- When mounting an NFS share, you'll need to ensure that Secure Transfer Required is disabled on the storage account. App Service doesn't support mounting NFS shares when this is enabled. It uses port 2409 and virtual network integration and private endpoints as the security measure.
```
